02/Evgeny Vasiukevich/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ChangeEvent, useState } from 'react';
+import HotelsBlock from './components/HomesGuestsLoves/HotelsBlock';
+import SearchForm from './components/UI/SearchForm/SearchForm';
+import { hotelsData } from './data/hotelsData';
+
+function App() {
+  const [filterHotels, setFilterHotels] = useState(hotelsData);
+  const [renderHotels, setRenderHotels] = useState(filterHotels);
+
+  const handleFilter = (event: ChangeEvent<HTMLInputElement>): void => {
+    setFilterHotels(hotelsData.filter(hotel => 
+      (hotel.name + hotel.city + hotel.country).toUpperCase().includes(event.target.value.toUpperCase())
```

Comment:
use template strings `${}`

Reply:
Do you mean for for brackets before .toUpperCase?
So?
`${hotel.name}${hotel.city}${hotel.country}`.toUpperCase()...

Reply:
yes